plugins/org.jkiss.dbeaver.tasks.ui.view/src/org/jkiss/dbeaver/tasks/ui/view/DatabaseTasksTree.java

```diff
@@ -568,6 +580,7 @@ private void addCheckedItem(TreeItem item, List<DBTTask> tasks) {
                 } else {
                     // Tasks
                     List<DBTTask> allTasksThisCategory = getSortedByCategoryTasks(parentCat);
+                    allTasksThisCategory.sort((o1, o2) -> o1.getName().compareToIgnoreCase(o2.getName()));
```

Comment:
```suggestion
                    allTasksThisCategory.sort(DBUtils.nameComparatorIgnoreCase());
```

Reply:
Is sorting here essential? We can define a custom comparator for the viewer instead.